rebasehelper/options.py

```diff
@@ -181,6 +181,13 @@
         "dest": "disable_inapplicable_patches",
         "help": "disable inapplicable patches in rebased SPEC file",
     },
+    {
+        "name": ["--keep-comments"],
+        "default": False,
+        "switch": True,
+        "dest": "keep_comments",
+        "help": "do not remove comments associated to patches when removing them",
```

Comment:
I think it should be `associated with`. But maybe it would be better to describe it as `do not remove associated comments when removing patches from spec file`.